CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ The following configuration changes in the `[data]` section may need to changed
 - [#7334](https://github.com/influxdata/influxdb/issues/7334): Panic with unread show series iterators during drop database
 - [#7482](https://github.com/influxdata/influxdb/issues/7482): Fix issue where point would be written to wrong shard.
 - [#7431](https://github.com/influxdata/influxdb/issues/7431): Remove /data/process_continuous_queries endpoint.
+- [#7053](https://github.com/influxdata/influxdb/issues/7053): Delete statement ignores a retention policy
```

Comment:
I think this should say something like "delete statement returns error when retention policy is specified" since that describes the fix rather than the problem.